libs/core/src/components/pds-button/docs/pds-button.mdx

```diff
@@ -157,7 +157,7 @@ Full width buttons are intended to be used when the button needs to take up the
   <pds-button variant="primary" full-width="true">Full Width</pds-button>
 </DocCanvas>
 
-## Loading
+### Loading
 
 The loading prop allows a `pds-button` to display a loading state.
```

Comment:
I think we have to check the loading when using slots. The icons are still visible.

Reply:
Thanks for catching that, should be good now.